application/src/main/kotlin/com/studentcenter/weave/application/chat/port/inbound/SaveChatMessage.kt

```diff
@@ -0,0 +1,9 @@
+package com.studentcenter.weave.application.chat.port.inbound
+
+import com.studentcenter.weave.domain.chat.entity.ChatMessage
+
+interface SaveChatMessage {
+
+    fun handleChatMessage(chatMessage: ChatMessage)
```

Comment:
p3) 메서드명 그냥 save하는 건 별로일까요? Handle보다 더 직관적인 것 같아서요. 그리고 이 친구는 invoke로 안하는 이유가 있나요?
```suggestion
    fun save(chatMessage: ChatMessage)
```

Reply:
아이건 네이밍을 잘못했네요, invoke라고 하는게 맞는거같아요